integration_cleanup/dns_cleanup.py

```diff
@@ -0,0 +1,126 @@
+import xmlrpc.client
+
+
+BATCH_SIZE = 100
+
+
+class DnsCleanupInstance:
```

Comment:
@giovannicimolin, can't you reuse the code in `gandi.py`?

Reply:
Just did this. I'll test it now.